menoh/menoh_example.cpp

```diff
@@ -103,27 +103,29 @@ int main(int argc, char** argv) {
     // dims of output is automatically calculated later
     menoh::variable_profile_table_builder vpt_builder;
     vpt_builder.add_input_profile(conv1_1_in_name, menoh::dtype_t::float_, {batch_size, channel_num, height, width});
-    vpt_builder.add_output_name(fc6_out_name);
+    // vpt_builder.add_output_name(fc6_out_name);
     vpt_builder.add_output_name(softmax_out_name);
 
     std::cout << "Build VPT..." << std::endl;
     // Build variable_profile_table and get variable dims (if needed)
     auto vpt = vpt_builder.build_variable_profile_table(model_data);
+    /*
     auto fc6_dims = vpt.get_variable_profile(fc6_out_name).dims;
```

Comment:
Ah, I realized you've removed this. It seems shufflenet already runs thanks to this change. However, it seems label mismatches so we should find a right label file to use shufflenet...

Reply:
Seems the label mismatch is caused by preprocess mistakes.
I'll check it